docroot/themes/humsci/humsci_basic/src/scss/partials/components/_pattern.carousel.scss

```diff
@@ -382,7 +382,7 @@
         // slick dots do not display until this breakpoint
         @media (min-width: 890px) {
           display: inline-block;
-          clip: inherit;
+          clip-path: inherit;
```

Comment:
Deprecated style that is now throwing warnings for the test:sass and lint:sass - https://developer.mozilla.org/en-US/docs/Web/CSS/clip